tests/parallel/independent.rs

```diff
@@ -8,11 +8,11 @@ fn in_par_two_independent_queries() {
     let mut db = ParDatabaseImpl::default();
 
     db.set_input('a', 100);
-    db.set_input('b', 010);
-    db.set_input('c', 001);
+    db.set_input('b', 10);
+    db.set_input('c', 1);
     db.set_input('d', 200);
-    db.set_input('e', 020);
-    db.set_input('f', 002);
```

Comment:
the formatting here was intentional, but...it's fine

Reply:
I agree it's not a big deal, but maybe 200, 220, 202?